### PR TITLE
SW-698 Fetch Shopware version from Shopware directly

### PR DESCRIPTION
**Pull request by TobiasGraml11**

### CODE REVIEW

File: src/Resources/config/services/services.xml [hunk 1]
```diff
@@ -157,7 +157,9 @@
         />
 
         <service id="FINDOLOGIC\FinSearch\Findologic\Api\FindologicSearchService" autowire="true" />
-        <service id="FINDOLOGIC\FinSearch\Findologic\Api\SortingService" autowire="true" />
+        <service id="FINDOLOGIC\FinSearch\Findologic\Api\SortingService" autowire="true">
+            <argument type="string" key="$shopwareVersion">%kernel.shopware_version%</argument>
```
Comment (dblaichinger) [inline src/Resources/config/services/services.xml:162]: Should there be a leading `$` for the key?

`key="$shopwareVersion"`

File: src/Resources/config/services/services.xml [hunk 2]
```diff
@@ -157,7 +157,9 @@
         />
 
         <service id="FINDOLOGIC\FinSearch\Findologic\Api\FindologicSearchService" autowire="true" />
-        <service id="FINDOLOGIC\FinSearch\Findologic\Api\SortingService" autowire="true" />
+        <service id="FINDOLOGIC\FinSearch\Findologic\Api\SortingService" autowire="true">
```
Comment (dblaichinger) [inline src/Resources/config/services/services.xml:161]: Maybe define the service outside of the `FindologicSearchService`?

**Review by dblaichinger — CHANGES_REQUESTED** [review]

File: src/Resources/config/services/services.xml (hunk 2 above)
Comment (TobiasGraml11) [reply src/Resources/config/services/services.xml:161]: What do you mean? This are defined as completely separate services here.

Comment (dblaichinger) [reply src/Resources/config/services/services.xml:161]: We can leave it as it is until the service need to be injected somewhere else. It was just a suggestion.

Comment (TobiasGraml11) [reply src/Resources/config/services/services.xml:161]: Can you elaborate on your suggestion? :P
I don't get the point at all

Comment (dblaichinger) [reply src/Resources/config/services/services.xml:161]: As example see `FindologicConfigService`, defined on line 23 and injected as argument in several other lines.

File: src/Resources/config/services/services.xml (hunk 1 above)
Comment (TobiasGraml11) [reply src/Resources/config/services/services.xml:162]: Yes:
https://symfony.com/doc/current/service_container.html#manually-wiring-arguments

File: src/Resources/config/services/services.xml (hunk 2 above)
Comment (dblaichinger) [reply src/Resources/config/services/services.xml:161]: As discussed in Slack, I read it wrong, but please add empty lines between those services.

File: src/Resources/config/services/services.xml (hunk 1 above)
Comment (dblaichinger) [reply src/Resources/config/services/services.xml:162]: Discussed in Slack, resolved.

**Review by dblaichinger — APPROVED** [review]

**Pull request merged**